### DISCUSSION TITLE
Welcome to google-maps-scraper Discussions!

### DISCUSSION THREAD

**Discussion by gosom** [gosom/google-maps-scraper#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shuhankuang** [comment]


**Reply by gosom** [reply]
🙏

**Comment by Mukesh-234234** [comment]
Hello I have a question! what does the depth option do?
